src/main/java/de/vogel612/testclient_javabot/client/custom/MessageBox.java

```diff
@@ -0,0 +1,36 @@
+package de.vogel612.testclient_javabot.client.custom;
+
+import javafx.geometry.Pos;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.text.Text;
+
+import com.gmail.inverseconduit.datatype.ChatMessage;
+
+public class MessageBox extends HBox {
+	
+	private Text name;
+	private Label message;
+	
+	public MessageBox(ChatMessage chatMessage) {
+		init();
+		name.setText(chatMessage.getUsername());
+		message = new Label(chatMessage.getMessage());
+		setLayout();
+		getChildren().addAll(name, message);
+	}
+	
+	private void init() {
+		name = new Text();
+	}
```

Comment:
Extracting this single line into a method seems overkill to me. Especially since it actually is the constructor's job to initialize fields. If something should be extracted it's things unrelated to fields (like name.setText() or setLayout())